src/drivers/utils/index.ts

```diff
@@ -39,3 +40,31 @@ export function createRequiredError(driver: string, name: string | string[]) {
   }
   return createError(driver, `Missing required option \`${name}\`.`);
 }
+
+export function getBinding(binding: KVNamespace | R2Bucket | string) {
```

Comment:
Can you please move this to `utils/cloudflare.ts`? 🙏🏼

Reply:
Sure, good idea. I have made changes :)